### PR TITLE
Edge native and Nested cluster.

### PR DESCRIPTION
**Pull request by nikchern**

### CODE REVIEW

File: spectrocloud/resource_cluster_edge_native.go [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+				ForceNew: true,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ssh_key": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"host": {
```
Comment (rishi-anand) [inline spectrocloud/resource_cluster_edge_native.go:156]: We should rename it to VIP

File: spectrocloud/resource_cluster_edge_native.go [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+							Optional: true,
+							Set:      schema.HashString,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+					},
+				},
+			},
+			"pack": {
```
Comment (rishi-anand) [inline spectrocloud/resource_cluster_edge_native.go:171]: We should remove pack in new resource, this is deprecated field.

File: spectrocloud/resource_cluster_edge_native.go [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+									},
+								},
+							},
+						},
+						"placements": {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"appliance_id": {
```
Comment (rishi-anand) [inline spectrocloud/resource_cluster_edge_native.go:292]: These values are for edge-virtual not for edge-native.

File: spectrocloud/resource_cluster_edge_native.go (hunk 1 above)
Comment (nikchern) [reply spectrocloud/resource_cluster_edge_native.go:157]: renamed

File: spectrocloud/resource_cluster_edge_native.go (hunk 2 above)
Comment (nikchern) [reply spectrocloud/resource_cluster_edge_native.go:172]: removed

File: spectrocloud/resource_cluster_edge_native.go (hunk 3 above)
Comment (nikchern) [reply spectrocloud/resource_cluster_edge_native.go:293]: cleanup is done

**Pull request merged**